.github/workflows/deploy-admin.yml

```
@@ -0,0 +1,82 @@
+
+    - name: Prepare deployment package
+      run: |
+        rm -rf deploy && mkdir deploy
+        cp application-admin/build/libs/application-admin-*.jar deploy/application-admin.jar
+        cp scripts/start-admin.sh scripts/stop-admin.sh appspec-admin.yml deploy/appspec.yml
+        cd deploy
+        zip -r ../${{ env.PROJECT_NAME }}-${{ env.DIVISION }}-${{ github.sha }}.zip .
+        cd ..
+    
+    - name: AWS credential 설정
+      uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)